### PR TITLE
fix device endpoint; add push_live script

### PR DESCRIPTION
**Pull request by liewegas**

### CODE REVIEW

**Comment by liewegas** [comment]
This is what's currently live (pushed with sudo ./push_live.sh)

File: push_live.sh [hunk 1]
```diff
@@ -0,0 +1,8 @@
+#!/bin/bash -ex
+
+TMP=/tmp/telemetry-push
```
Comment (alfredodeza) [inline push_live.sh]: this would benefit from using `mktemp -d`

File: push_live.sh [hunk 2]
```diff
@@ -0,0 +1,8 @@
...
+
+rm -rf $TMP
```
Comment (alfredodeza) [inline push_live.sh]: when removing paths, better to use quotes always, even though this knows the path is not going to have sneaky whitespace

File: tables.txt [hunk 3]
```diff
@@ -7,7 +7,7 @@ CREATE TABLE report (
 );
 
 CREATE TABLE device_report (
-       device_id VARCHAR(50),
+       device_id VARCHAR(128),
```
Comment (alfredodeza) [inline tables.txt:10]: does this mean that the DB is going to need recreation?

File: push_live.sh (hunk 2 above)
Comment (dmick) [reply push_live.sh]: agreed, but, this is a crappy little nothing of a script for such care.  I know, I know.

File: push_live.sh (hunk 1 above)
Comment (dmick) [reply push_live.sh]: agreed, see next comment

File: tables.txt (hunk 3 above)
Comment (dmick) [reply tables.txt:10]: probably.  there's no data in it yet, though.

**Review by dmick — APPROVED** [review]

**Pull request merged**